ppc64.risu

```diff
@@ -3156,11 +3199,6 @@ XVCMPGTSP PPC64LE 111100 t:5 a:5 b:5 01001011 ax:1 bx:1 tx:1
 # format:XX3 book:I page:672 v2.06 xvcmpgtsp[.] VSX Vector Compare Greater Than SP
 XVCMPGTSPd PPC64LE 111100 t:5 a:5 b:5 11001011 ax:1 bx:1 tx:1
 
-# format:XX3 book:I page:673 v3.0 xvcmpnedp[.] VSX Vector Compare Not Equal Double-Precision
-XVCMPNEDP PPC64LE 111100 t:5 a:5 b:5 01111011 ax:1 bx:1 tx:1
-# format:XX3 book:I page:673 v3.0 xvcmpnedp[.] VSX Vector Compare Not Equal Double-Precision
-XVCMPNEDPd PPC64LE 111100 t:5 a:5 b:5 11111011 ax:1 bx:1 tx:1
-
```

Comment:
So, I just noticed I sent this in this patch.

The problem with these lines is that theses instructions were removed from Power ISA. I think they were introduced in 3.0 and removed in 3.0B. Do you think we should remove them from risu?
If we remove them, I'll have to send a separated patch, because it should not be in this commit

Reply:
This is also the case for LMDX, it was present in the first version of PowerISA v3.0 but removed on PowerISA v3.0b. I think we can keep them commented out, just like we do with other instructions that cannot be tested like SC[V], TRAP, etc.